src/App.jsx

```diff
@@ -1,7 +1,16 @@
+import { Header } from "./components/Header.jsx";
+import { Album } from "./components/Album.jsx";
 import data from "./data.json";
+import "./components/styling/app.css"
 
-console.log(data);
-
+//pass json data to  *mArray* in Album
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
-};
+
+  return (
+    <div className="app-wrap">
+      <Header albumList={data} />
+      <Album mArray={data} />
```

Comment:
what does mArray stand for?

Reply:
music array :)

Reply:
Valid question Trista! Maybe it could be named albums or something more descriptive? 😉

Reply:
wanted to avoid confusion with Album component with the same name for album (data) - but I see your point(s) its not a good name.